hypothesis-python/src/hypothesis/strategies/_internal/core.py

```diff
@@ -120,6 +120,34 @@
     Protocol = object  # type: ignore[assignment]
 
 
+try:
+    import typing_extensions
+except ImportError:
+    typing_extensions = None  # type: ignore
+
+
+RequiredTypes: tuple = ()
+try:
+    RequiredTypes += (typing.Required,)
```

Comment:
`mypy` quite reasonably wants a `# type: ignore` here, since the attribute might not exist (and it doesn't check for the exception handling); same below for `NotRequired`.

Reply:
got it